src/MoBi.Presentation/Tasks/Interaction/InteractionTasksForPathAndValueEntity.cs

```diff
@@ -184,10 +184,19 @@ protected IMoBiCommand EditFormulaAndSetOnBuilder(TBuildingBlock buildingBlock,
 
             if (!modalPresenter.Show())
                return new MoBiEmptyCommand();
-               
-            macroCommand.Add(_interactionTaskContext.MoBiFormulaTask.UpdateFormula(builder, builder.Formula, editFormulaPresenter.Formula, usingFormulaDecoder, buildingBlock));
-            macroCommand.Add(_interactionTaskContext.MoBiFormulaTask.AddFormulaToCacheOrFixReferenceCommand(buildingBlock, builder).RunCommand(_interactionTaskContext.Context));
-            macroCommand.Add(setValue(builder, null, builder.DisplayUnit, buildingBlock));
+
+            var newFormula = editFormulaPresenter.Formula;
+            if (newFormula is ConstantFormula constantFormula)
+            {
+               macroCommand.Add(setValue(builder, constantFormula.Value, constantFormula.Dimension.BaseUnit, buildingBlock));
```

Comment:
is it correct, that the **base unit** and not the **display unit** is passed to `setValue()`?